### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by hafabenny**
carousel not showing up in storybook.  could you help investigate?
added a couple of notes of what I believe sets a slide "active" and visible.

### CODE REVIEW

File: lib/carousel/carousel-item.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        in={isIn}
+        onEnter={this.onEnter}
+        onEntering={this.onEntering}
+        onExit={this.onExit}
+        onExiting={this.onExiting}
+        onExited={this.onExited}
+      >
+        {(status) => {
+          const { direction } = this.context;
+          const isActive =
```
[the remark this answers is not in the document]
Comment (bwarner) [reply lib/carousel/carousel-item.js]: There is an active class and a active pseudo class. I list group is using the pseudo class, carouselItem is using the class active from bootstrap.

File: lib/carousel/carousel-item.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+              || status === TransitionStatuses.EXITING)
+            && startAnimation
+            && (direction === 'right' ? carouselItemLeft : carouselItemRight);
+          const orderClassName =
+            status === TransitionStatuses.ENTERING
+            && (direction === 'right' ? carouselItemNext : carouselItemPrev);
+          const itemClasses = cx(
+            className,
+            carouselItem,
+            isActive && active,
```
[the remark this answers is not in the document]
Comment (bwarner) [reply lib/carousel/carousel-item.js]: take active out of the call to cx, it not working right when we put conditional expression together.  Maybe we aren't using cx right, but for now I'm putting them in the actual className attribute.

**Comment by hafabenny** [comment]
when you say animation is off. do you mean between the indicator dashes and the each slide coming into view?  or the animation is "janky" in general?

File: tests/__snapshots__/storyshots.test.js.snap [hunk 3]
```diff
@@ -7738,7 +7738,7 @@ exports[`Image storyshots Components/Carousel Example 1`] = `
         role="listbox"
       >
         <div
-          className="emotion-3 emotion-0"
+          className="emotion-3 emotion-0  "
```
Comment (bwarner) [inline tests/__snapshots__/storyshots.test.js.snap]: you have some extra spaces somewhere

**Pull request merged**